README.md

```diff
@@ -283,6 +283,55 @@ After the version is printed, the container is stopped and removed:
 docker run --rm -it ghcr.io/cycrilabs/keycloak-configurator:latest -V
 ```
 
+### Running with docker-compose
+
+In case of configuring existing keycloak container in docker-compose. This can be done via this sample setup
+
+```docker-compose
+services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:23.0.0
+    environment:
+      KC_HTTP_RELATIVE_PATH: auth
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+    command: ["start-dev", "--http-relative-path=auth", "--http-port=5050", "--import-realm","--hostname=localhost", "--hostname-port=5050"]
+    ports:
+      - 5050:5050
+    volumes:
+      - ./../keycloak/setup:/opt/keycloak/data/import:Z
+
+  postgres:
+    image: postgres:16-alpine
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+    volumes:
+      - ./postgresql/postgres-init-user-db.sql:/docker-entrypoint-initdb.d/init-users.sql:Z
+      
+  keycloak-configurator:
+    image: ghcr.io/cycrilabs/keycloak-configurator:latest
+    environment:
```

Comment:
All environment variables of the configurator can be removed. They won't be used.